sources/platform/actors/development/builds_and_runs/builds.md

```diff
@@ -1,34 +1,65 @@
+
+### Build numbers
+
+Each build is assigned a unique build number if the format _MAJOR\.MINOR\.BUILD_ (e.g. _1\.2\.345_):
+
+- _MAJOR\.MINOR_ corresponds to the Actor version number
+- _BUILD_ is an automatically incremented number starting at **1**.
+
+### Build resources
+
+By default, builds have the following resource allocations:
 
-Each build is assigned a unique build number of the form **MAJOR\.MINOR\.BUILD** (e.g. **1\.2\.345**), where **MAJOR\.MINOR** corresponds to the Actor version number (see [Versioning](#versioning)), and **BUILD** is an automatically-incremented number starting at **1**.
+- Timeout of _1800_ seconds
+- Memory: `4096 MB`
 
-By default, the build has a timeout of 300 seconds and consumes 4096 MB (2048 MB on the free plan) of memory from the user's memory limit. See the [Resource limits](../../running/index.md) section for more details.
+Check out [Resource limits](../../running/index.md) section for more details.
 
 ## Versioning
 
-In order to enable active development, the Actor can have multiple versions of the source code and associated settings, such as the **Base image** and **Environment**. Each version is denoted by a version number of the form `MAJOR.MINOR`; the version numbers should adhere to the [Semantic Versioning](https://semver.org/) logic.
+To support active development, Actors can have multiple versions versions of source code and associated settings, such as the base image and environment. Each version is denoted by a version number of the form _MAJOR.MINOR_; following [Semantic Versioning](https://semver.org/) principles.
 
-For example, the Actor can have a production version **1.1**, a beta version **1.2** that contains new features but is still backward compatible, and a development version **2.0** that contains breaking changes.
+For example, an Actor might have:
+
+- Production version _1.1_
+- Beta version _1.2_ that contains new features but is still backward compatible
+- Development version _2.0_ that contains breaking changes.
 
 ## Tags
 
-When running the Actor, the caller needs to specify which build should actually be used. To simplify this process, the builds can be associated with a tag such **latest** or **beta**, which can be used instead of the version number when running the Actor. The tags are unique - only one build can be associated with a specific tag.
+Tags simplify the process of specifying which build to use when running an Actor Instead of using a version number, you can use a tag such as _latest_ or _beta_. Tags are unique, meaning only one build can be associated with a specific tag.
```

Comment:
```suggestion
Tags simplify the process of specifying which build to use when running an Actor. Instead of using a version number, you can use a tag such as _latest_ or _beta_. Tags are unique, meaning only one build can be associated with a specific tag.
```